bin/signalk-server

```diff
@@ -16,6 +16,16 @@
  * limitations under the License.
 */
 
+const { engines } = require('../package.json');
+const semver = require('semver');
+
+const version = engines.node;
+
+if (!semver.satisfies(process.version, version)) {
+  console.log(`The installed version of node (${process.version}) is older than the required minimum version (${version}). See https://github.com/SignalK/signalk-server/wiki/Updating-to-Node.js-18 for more information how to upgrade.`);
```

Comment:
I reorganised the wiki, this should point to non version specific instructions at https://github.com/SignalK/signalk-server/wiki/Installing-and-Updating-Node.js